### PR TITLE
RSS Menu Lans to Generic Terms #6

### PR DESCRIPTION
**Pull request by MikeyGMT**
Replace duplicates with common terms in RSS Plugin

### CODE REVIEW

**Comment by MikeyGMT** [comment]
Testing 

image

image

image

image

File: e107_plugins/rss_menu/admin_prefs.php [hunk 1]
```diff
@@ -94,19 +94,19 @@ class rss_ui extends e_admin_ui
...
-		  'rss_path'        =>   array ( 'title' => 'Plugin-Dir',   'type' => 'text', 'data' => 'str', 'readonly'=>1, 'width' => 'auto', 'help' => '', 'readParms' => '', 'writeParms' => '', 'class' => 'left', 'thclass' => 'left',  ),
+		  'rss_path'        =>   array ( 'title' => LAN_PLUGIN_FOLDER,'type' => 'text', 'data' => 'str', 'readonly'=>1, 'width' => 'auto', 'help' => '', 'readParms' => '', 'writeParms' => '', 'class' => 'left', 'thclass' => 'left',  ),
 		  'rss_url'         =>   array ( 'title' => LAN_URL,        'type' => 'method', 'data' => 'str', 'width' => 'auto', 'help' => '', 'readParms' => '', 'writeParms' => '', 'class' => 'left', 'thclass' => 'left',  ),
-		  'rss_topicid'     =>   array ( 'title' => 'Topic id',     'type' => 'text', 'data' => 'int', 'width' => 'auto', 'help' => '', 'readParms' => '', 'writeParms' => '', 'class' => 'center', 'thclass' => 'center',  ),
+		  'rss_topicid'     =>   array ( 'title' => LAN_ID,         'type' => 'text', 'data' => 'int', 'width' => 'auto', 'help' => '', 'readParms' => '', 'writeParms' => '', 'class' => 'center', 'thclass' => 'center',  ),
```
Comment (CaMer0n) [inline e107_plugins/rss_menu/admin_prefs.php]: I believe this needs to be Topic Id, otherwise we have 2 fields with the same title.

**Comment by CaMer0n** [comment]
Thank you @MikeyGMT for this! 👍 (please see small issue in comment above)

File: e107_plugins/rss_menu/admin_prefs.php (hunk 1 above)
Comment (MikeyGMT) [inline e107_plugins/rss_menu/admin_prefs.php]: Ooo OK, will add Topic ID

**Comment by MikeyGMT** [comment]
Fixed 
image

image

Not sure I understand how code climate assumes we can never get to this?  

> The method dboptions uses an else expression. Else is never necessary and you can simplify the code to work without else.

NEW

```
        {
            return LAN_NOCHANGE_NOTSAVED;
        }
```

**Pull request merged**